docs/feature_split_keyboard.md

```diff
@@ -130,24 +130,31 @@ To enable this method, add the following to your `config.h` file:
 #define EE_HANDS
 ```
 
-However, you'll have to flash the EEPROM files for the correct hand to each controller.  You can do this manually, or there are targets for avrdude and dfu to do this, while flashing the firmware: 
-
-* `:avrdude-split-left`
-* `:avrdude-split-right`
-* `:dfu-split-left`
-* `:dfu-split-right`
-* `:dfu-util-split-left`
-* `:dfu-util-split-right`
+Next, you will have to flash the EEPROM files once for the correct hand to the controller on each halve. You can do this manually with the following bootloader targets while flashing the firmware:
```

Comment:
For controllers using emulated eeprom these need to be specified every time you flash.

Reply:
> For controllers using emulated eeprom these need to be specified every time you flash.

I presume this apply to ARM controllers?

Reply:
Yeah a few of them anyway.

Reply:
Added a note to cover this fact in the ARM controller section that follows.